src/test/components/CallNodeContextMenu.test.js

```diff
@@ -41,70 +41,68 @@ describe('calltree/CallNodeContextMenu', function() {
   }
 
   function setup(store = createStore(), forceOpenForTests = true) {
-    const wrapper = mount(
+    const renderResult = render(
       <Provider store={store}>
         <CallNodeContextMenu forceOpenForTests={forceOpenForTests} />
       </Provider>
     );
 
-    function findMenuItem(type: string) {
-      return wrapper.find(MenuItem).findWhere(n => {
-        const { data } = n.props();
-        return typeof data === 'object' && data.type === type;
-      });
-    }
-
-    return { store, wrapper, findMenuItem };
+    return { ...renderResult, getState: store.getState };
   }
 
   describe('basic rendering', function() {
     it('renders a blank context menu (with only 1 div) before it is open', () => {
       const isContextMenuOpen = false;
-      const { wrapper } = setup(createStore(), isContextMenuOpen);
-      expect(wrapper.find('ContextMenu > nav').children().length).toBe(1);
-      expect(wrapper).toMatchSnapshot();
+      const { container } = setup(createStore(), isContextMenuOpen);
+      expect(
+        ensureExists(container.querySelector('.react-contextmenu')).children
+      ).toHaveLength(1);
+      expect(container.firstChild).toMatchSnapshot();
     });
 
     it('renders a full context menu when open, with many nav items', () => {
       const isContextMenuOpen = true;
-      const { wrapper } = setup(createStore(), isContextMenuOpen);
+      const { container } = setup(createStore(), isContextMenuOpen);
       expect(
-        wrapper.find('ContextMenu > nav').children().length > 1
+        ensureExists(container.querySelector('.react-contextmenu')).children
+          .length > 1
       ).toBeTruthy();
-      expect(wrapper).toMatchSnapshot();
+      expect(container.firstChild).toMatchSnapshot();
     });
   });
 
   describe('clicking on call tree transforms', function() {
     // Iterate through each transform slug, and click things in it.
-    for (const type of [
-      'merge-call-node',
-      'merge-function',
-      'focus-subtree',
-      'focus-function',
-      'collapse-function-subtree',
-      'collapse-resource',
-      'collapse-direct-recursion',
-      'drop-function',
-    ]) {
+    const fixtures = [
+      { matcher: /Merge node/, type: 'merge-call-node' },
+      { matcher: /Merge function/, type: 'merge-function' },
+      { matcher: /Focus.*subtree/, type: 'focus-subtree' },
+      { matcher: /Focus.*function/, type: 'focus-function' },
+      { matcher: /Collapse.*subtree/, type: 'collapse-function-subtree' },
+      { matcher: /Collapse functions/, type: 'collapse-resource' },
+      { matcher: /Collapse.*recursion/, type: 'collapse-direct-recursion' },
+      { matcher: /Drop samples/, type: 'drop-function' },
+    ];
```

Comment:
I decided to go by such a structure to make it more "user-like", by finding the menu item from its label.